payments-core/res/layout/stripe_card_input_widget.xml

```diff
@@ -3,7 +3,7 @@
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools">
 
-    <com.stripe.android.view.CardBrandView
+    <com.stripe.android.view.CardBrandViewRebuild
```

Comment:
if I understand correctly, this implies that this PR switches the implementation to use the new view -- right? If so, should we hold off on merging this until we have tests and have done a bug bash?

Reply:
I retargeted the PR to feature branch `tyler/wrapped-adhere-feature`. That way we can merge once this is approved, do a bug bash, and once ready to merge I'll update the feature branch to remove the rebuild suffices and fully remove the compose CBC dropdown, and then merge that to master

Reply:
sounds good! Thanks!